.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: java
+
+services: docker
+jobs:
+    include:
+        - stage: Debricked-scan
+          on:
+            branch: "*"  
+          env:
+          -  DEBRICKED_TOKEN=${DEBRICKED_TOKEN}
+
+          before_install: 
+          -  env > file.env  
+          -  ./gradlew dependencies > .debricked-gradle-dependencies.txt
+
+          #https://docs.travis-ci.com/user/languages/java/#projects-using-gradle
+          before_cache:
+            - rm -f $HOME/.gradle/caches/modules-2/modules-2.lock
```

Comment:
Looking at the URL you put in the template @tamarailenacosta, it seems like you have excluded
`- rm -fr $HOME/.gradle/caches/*/plugin-resolution/` from `before_cache:`. 
Is there a reason for it?